server/apps/archive/archive_spike.py

```diff
@@ -78,13 +78,14 @@ def on_update(self, updates, original):
     def _validate_item(self, original):
         """
         Raises an exception if the item is linked in a non-take package, the idea being that you don't whant to
-        inadvertently remove thing from packages, this force that to be done as a conscious action.
+        inadvertently remove the item from the packages, this force that to be done as a conscious action.
         :param original:
         :raise: An exception or nothing
         """
         if is_item_in_package(original):
-            raise SuperdeskApiError.badRequestError(message="This item is in a package" +
-                                                            " it needs to be removed before the item can be spiked")
+            raise SuperdeskApiError.badRequestError(
```

Comment:
The best readability would IMO be achieved if the error message is extracted into its own variable, which would then be passed as an argument to the `badRequestError()` method, though it's not really that big of a deal.